*Creado por:*

*Isabel Maniega*

# Ejemplos de Estandarización

## Ejemplo 1

La altura promedio de un jugador de baloncesto profesional fue de 2.00 metros con una desviación estándar de 0.02 metros. Harrison Barnes es un jugador de baloncesto que mide 2.03. ¿ Cuántas desviaciones estándar de la media es la altura de Barnes?

*Primero debemos dibujar la curva normal que representa la distribución de las alturas de los jugadores de baloncesto.*

<img src="ejer_1.png">

*Observa que colocamos la altura media 2.00 justo en el medio de la distribución y hacemos marcas que están a 1 desviación estandar o 0.02 metros de distancia en ambas direcciones*

*A continuación, debemos calcular la puntuación estándar (es decir, la puntuación Z) para la altura de Barnes. Ya que µ= 2.00, s= 0.02 y x = 2.03 podemos encontrar el puntaje Z.*

In [1]:
# Z score = x - µ / s
Zscore = round((2.03-2)/0.02, 2)
Zscore

1.5

In [2]:
# Otra forma

µ = 2.0
x = 2.03
s = 0.02

In [3]:
Zscore = round((x - µ)/s, 2)
Zscore

1.5

Encontrarmos 1.5 como la puntuación Z, nos dice que la altura de Barnes está 1.5 desviaciones estándar de la media, es decir, 1.5 * s + µ = Altura de Barnes

In [4]:
Altura_Barnes = (Zscore * s) + µ
Altura_Barnes

2.03

## Ejemplo 2

La altura de promedio de un jugador de hockey profesional es de 1.86 metros con una desviación estándar de 0.06 metros. Tuler Myers, un profesional de hockey, tiene la misma altura que Harrison Barnes. ¿Cuál de los dos es más alto en su respectiva liga?

*Para encontrar la puntuación estándar de Tyler Myers podemos usar la información: µ= 1.86, s=0.06, x= 2.03. Esto da como resultado la puntuación estándar:*

In [5]:
µ = 1.86
s = 0.06
x = 2.03

In [6]:
Zscore = round((x - µ)/ s, 2)
Zscore

2.83

*Comparando las dos puntuaciones Z, vemos que la puntuación de Tyler Myers es de 2.83 es mayor que la puntuación de Barnes 1.5. Esto nos dice que hay más jugadores de hockey más bajos que Myers que jugadores de baloncesto más bajos que Barnes.*

## Ejercicios

### Ejercicio 1:

Calcula la Z score para los siguientes casos:

* µ = 54, s = 12, x = 68
* µ = 25, s = 3.5, x = 20
* µ = 0.01, s = 0.002, x = 0.01

In [7]:
def calculo_z(u, s,x):
    return round((x - u)/s, 2) 
    

In [8]:
u = 54
s = 12
x = 68

Zscore = round((x - u)/ s, 2)
Zscore

1.17

In [9]:
calculo_z(u,s,x)

1.17

In [10]:
µ = 25
s = 3.5 
x = 20
Zscore2 = round((x - µ)/ s, 2)
Zscore2

-1.43

In [17]:
calculo_z(25,3.5,20)

-1.43

In [19]:
µ = 0.01
s = 0.002 
x = 0.01
Zscore3 = round((x - µ)/ s, 5)
Zscore3

0.0

In [20]:
calculo_z(µ,s,x)

0.0

### Ejercicio 2:

El GPA promedio de los estudiantes en una escuela de secundaria local es 3.2 con una desviación estándar de 0.3. Jenny tiene GPA de 2.8 ¿A cuántas desviaciones estándar de la media está el GPA de Jenny?

In [12]:
µ = 3.2
x = 2.8
s =0.3
ZscoreJenny = round((x - µ)/ s, 2)
ZscoreJenny

-1.33

### Ejercicio 3:

Jenny está tratando de demostrarles a sus padres que le va mejor en la escuela que a su prima. Su prima asiste a una escuela de secundaria diferente donde el GPA promedio es 3.4 con una desviación estándar de 0.2. El prima de Jenny tiene un GPA de 3.0.¿se está desempeñando Jenny mejor que su prima según los puntajes estándar?

In [13]:
µ = 3.4
x = 3
s =0.2
ZscorePrima = round((x - µ)/ s, 2)
ZscorePrima

-2.0

### Ejercicio 4:

La puntuación de Kyle en una prueba de matemáticas reciente fue de 2.3 desviaciones estándar de la puntuación media del 78%. Si la desviación estándar de los puntajes de la prueba fue del 8%. ¿Qué puntaje obtuvo kyle en su prueba?

In [ ]:
media= 0.78 --- 78
desviacio=2.3
deviacion=0.08 --8


In [22]:
78+(2.3*8)

96.4

In [23]:
78-(2.3*8)

59.6

*Creado por:*

*Isabel Maniega*